docs/guides/packaging/pruning-chunks.mdx

```diff
@@ -28,16 +30,54 @@ A few empty chunks will make no significant difference in map size.
 
 ### 3. Click "Chunk Control"
 
-Click the Chunk Control button on the bottom right.
+Click the `Chunk Control` button on the bottom right.
 
-`Note:` You must be in "Chunk View" in order for this to work.`
+`Note:` You must be in "Chunk View" in order for this to work.
 
 ![Chunk Control Process](https://i.imgur.com/WXdnHn4.png)
 
 ### 4. Prune and save
 
 Now you can prune the map by clicking the `Prune` button in the sidebar on the left.
-That’s it! Your map is now pruned. Save with `CTRL+S`, then you can close MCEdit.
-Be sure to remove any folders or files generated by MCEdit.
+That's it! Your map is now pruned. Save with `CTRL` + `S`, then you may close MCEdit.
+Be sure to delete any folders or files [generated by MCEdit](/docs/guides/packaging/cleaning-files).
 
 ![Save Button](https://i.imgur.com/IDyKsoh.png)
+
+## Using MCASelector
+
+### 1. Download MCASelector
+
+MCASelector is an alternative program that makes chunk pruning simple compared to MCEdit.
+It is available for Windows, macOS, and Linux as well as supporting multiple versions of Minecraft.
+You also have the ability to load a Minecraft world with either a `level.dat` file or the `region` folder.
+Start by downloading MCASelector from the [GitHub page](https://github.com/Querz/mcaselector).
+If you are having trouble running MCASelector, you can consult the [installation guide](https://github.com/Querz/mcaselector/wiki/Installation) or [report an issue](https://github.com/Querz/mcaselector/issues).
+Additionally, it is a good idea to create a backup of the world as MCASelector will automatically save any changes you make.
+
+`Note:` For simplicity, a 16x16 chunk will be referred to as a chunk while a 32x32 chunk will be referred to as a region.
+You can toggle between each modes by zooming in or out while viewing a map.
+
+![MCASelector Download](https://i.imgur.com/3aTpkTY.png)
```

Comment:
I would remove this image since it's pretty large and there isn't a better more fitting equivalent to the mcedit screenshot.